contracts/exchangeIssuance/ExchangeIssuanceLeveragedForCompound.sol

```diff
@@ -0,0 +1,1346 @@
+
+    IController public immutable setController;
+    IDebtIssuanceModule public immutable debtIssuanceModule;
+    ICompoundLeverageModule public immutable compoundLeverageModule;
+    DEXAdapter.Addresses public addresses;
+    address public immutable cEtherADDRESS;
+
+    mapping(address => address) public cTokenPairs;
+
+    /* ============ Events ============ */
+
+    event ExchangeIssue(
+        address indexed _recipient,     // The recipient address of the issued SetTokens
+        ISetToken indexed _setToken,    // The issued SetToken
+        address indexed _inputToken,    // The address of the input asset(ERC20/ETH) used to issue the SetTokens
+        uint256 _amountInputToken,      // The amount of input tokens used for issuance
+        uint256 _amountSetIssued        // The amount of SetTokens received by the recipient
+    );
+
+    event ExchangeRedeem(
+        address indexed _recipient,     // The recipient address which redeemed the SetTokens
+        ISetToken indexed _setToken,    // The redeemed SetToken
+        address indexed _outputToken,   // The address of output asset(ERC20/ETH) received by the recipient
+        uint256 _amountSetRedeemed,     // The amount of SetTokens redeemed for output tokens
+        uint256 _amountOutputToken      // The amount of output tokens received by the recipient
+    );
+
+    /* ============ Modifiers ============ */
+
+    modifier onlyLendingPool() {
+        require(msg.sender == address(LENDING_POOL), "ExchangeIssuance: LENDING POOL ONLY");
+        _;
+    }
+
+    modifier isValidPath(
+        address[] memory _path,
+        address _inputToken,
+        address _outputToken
+    )
+    {
+        if(_inputToken != _outputToken){
+            require(
+                _path[0] == _inputToken || (_inputToken == addresses.weth && _path[0] == DEXAdapter.ETH_ADDRESS),
+                "ExchangeIssuance: INPUT_TOKEN_NOT_IN_PATH"
+            );
+            require(
+                _path[_path.length-1] == _outputToken ||
+                (_outputToken == addresses.weth && _path[_path.length-1] == DEXAdapter.ETH_ADDRESS),
+                "ExchangeIssuance: OUTPUT_TOKEN_NOT_IN_PATH"
+            );
+        }
+        _;
+    }
+
+
+    /* ============ Constructor ============ */
+
+    /**
+    * Sets various contract addresses
+    *
+    * @param _weth                  Address of wrapped native token
+    * @param _quickRouter           Address of quickswap router
+    * @param _sushiRouter           Address of sushiswap router
+    * @param _uniV3Router           Address of uniswap v3 router
+    * @param _uniV3Quoter           Address of uniswap v3 quoter
+    * @param _setController         SetToken controller used to verify a given token is a set
+    * @param _debtIssuanceModule    DebtIssuanceModule used to issue and redeem tokens
+    * @param _compoundLeverageModule    CompoundLeverageModule to sync before every issuance / redemption
```

Comment:
Nit:  columns of the param documentation not aligned.

Reply:
Seems resolved by 2bc43dfd123d9c826c22e1931048af595bb3ebc3